core/src/main/java/com/spotify/metrics/core/SemanticMetricBuilder.java

```diff
@@ -44,17 +44,8 @@ public boolean isInstance(final Metric metric) {
         }
     };
 
-    SemanticMetricBuilder<Histogram> HISTOGRAMS = new SemanticMetricBuilder<Histogram>() {
-        @Override
-        public Histogram newMetric() {
-            return new Histogram(new ExponentiallyDecayingReservoir());
-        }
-
-        @Override
-        public boolean isInstance(final Metric metric) {
-            return Histogram.class.isInstance(metric);
-        }
-    };
+    SemanticMetricBuilder<Histogram> HISTOGRAMS = SemanticMetricBuilderFactory
```

Comment:
Are there tests for this and timers already? if not pls add some.

Reply:
I already added tests for the `SemanticMetricBuilderFactory`. What kind of tests do we want to see with these? The builders are typed constants and the metrics themselves already have tests in codahale/dropwizard

Reply:
I was thinking along the lines of testing this...

https://github.com/spotify/semantic-metrics/blob/d78c9f066a81069840d5d1b7d761d1296e17ca31/core/src/main/java/com/spotify/metrics/core/SemanticMetricRegistry.java#L181

which those methods `.timer` and `.histogram` are not being tested directly in https://github.com/spotify/semantic-metrics/blob/master/core/src/test/java/com/spotify/metrics/core/SemanticMetricRegistryTest.java

Reply:
Ah, I see. Still, I think that testing those doesn't give us any value that the thorough testing of `getOrAdd` hasn't already given us,